README.md

```diff
@@ -67,3 +67,17 @@ public pull request.
 
 Pulls the upstream repository into the local repository.
 
+## [`openlane_run`](./openlane_run)
+
+The goal of this action is to run a given design through the OpenLane flow
+using a given tag. Optionally a PR can be created to update a stable tag
+file stored in the repository.
+Arguments:
+-   `ol_tag` [optional, default: `latest`]: OpenLane tag. You can set to
+    `ol_tag_file` to use the value from the input.
+-   `update_tag` [optional, default: `false`]: if the test is successful
+    update stable tag pointed by the `ol_tag_file` input.
+-   `ol_tag_file` [optional, default: `.github/openlane-stable-tag`]: File
```

Comment:
would the action fail if the file doesn't exist or create a PR?

Reply:
Fixed on [0b11b71](https://github.com/The-OpenROAD-Project/actions/pull/11/commits/0b11b7106247798cb27386930a42419f6d632d64)